bindings/python/iota_sdk/types/output.py

```diff
@@ -68,3 +87,11 @@ def from_dict(cls, dict: Dict) -> OutputWithMetadata:
         for k, v in dict.items():
             setattr(obj, k, v)
         return obj
+
+    def as_dict(self):
+        config = dict()
+
+        config['metadata'] = self.metadata.__dict__
```

Comment:
this means metadata still needs to be done right?

Reply:
No, but metadata has only "simple" fields, so __dict__ does the job and no extra method is needed